CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -179,28 +171,17 @@ bool AlCaRecoTriggerBitsRcdUpdate::addTriggerLists(const std::vector<edm::Parame
 }
 
 ///////////////////////////////////////////////////////////////////////
-void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(AlCaRecoTriggerBits *bitsToWrite) const {
+void AlCaRecoTriggerBitsRcdUpdate::writeBitsToDB(const std::unique_ptr<AlCaRecoTriggerBits> &bitsToWrite) const {
   edm::LogInfo("") << "Uploading to the database...";
 
   edm::Service<cond::service::PoolDBOutputService> poolDbService;
   if (!poolDbService.isAvailable()) {
     throw cms::Exception("NotAvailable") << "PoolDBOutputService not available.\n";
   }
 
-  // ownership of bitsToWrite transferred
   // FIXME: Have to check that timetype is run number! How?
```

Comment:
Does this comment still refer to an open issue, or can it be removed?

Reply:
This is not needed, I have checked that the default timetype is "Run" so I will remove this comment in my next commit.